ols/app/models/config.py

```diff
@@ -383,11 +380,59 @@ def validate_yaml(self) -> None:
         self.logging_config.validate_yaml()
 
 
+class DevConfig(BaseModel):
+    """Developer-mode-only configuration options."""
+
+    enable_dev_ui: bool = False
+    disable_question_validation: bool = False
+    llm_temperature_override: int | None = None
```

Comment:
Should llm_temperature_override be a float?

Reply:
apparently so, thanks :facepalm:

Reply:
fixed